index.js

```diff
@@ -36,6 +36,8 @@ function WriteStreamAtomic (path, options) {
   }
   Writable.call(this, options)
 
+  this.__isWin = (options && options.isWin) || (/^win/).test(process.platform)
```

Comment:
This is incorrect in two ways. If someone set `{isWin: false}` it will still try to detect.

```js
options = options || {};
this.__isWin = typeof options.isWin === 'boolean' ? options.isWin : (process.platform === 'win32');
```


The `platform` is always `win32` even on 64-bit.